app/src/main/java/com/rob729/quiethours/Activity/IntroScreen.kt

```diff
@@ -0,0 +1,49 @@
+package com.rob729.quiethours.Activity
+
+import android.content.Intent
+import android.os.Bundle
+import androidx.core.content.ContextCompat
+import androidx.fragment.app.Fragment
+import com.github.appintro.AppIntro
+import com.github.appintro.AppIntroCustomLayoutFragment
+import com.github.appintro.AppIntroPageTransformerType
+import com.rob729.quiethours.R
+import com.rob729.quiethours.util.AppConstants
+import com.rob729.quiethours.util.StoreSession
+
+class IntroScreen : AppIntro() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        setColorDoneText(ContextCompat.getColor(this, R.color.textColor))
+        setColorSkipButton(ContextCompat.getColor(this, R.color.textColor))
+        setNextArrowColor(R.color.textColor)
+        setIndicatorColor(
+            selectedIndicatorColor = ContextCompat.getColor(this, R.color.textColor),
+            unselectedIndicatorColor = ContextCompat.getColor(this, R.color.unSelected)
+        )
+        setTransformer(AppIntroPageTransformerType.SlideOver)
+        setImmersiveMode()
+
+        addSlide(AppIntroCustomLayoutFragment.newInstance(R.layout.intro_one))
+        addSlide(AppIntroCustomLayoutFragment.newInstance(R.layout.intro_two))
+        addSlide(AppIntroCustomLayoutFragment.newInstance(R.layout.intro_three))
+    }
+
+    override fun onSkipPressed(currentFragment: Fragment?) {
+        super.onSkipPressed(currentFragment)
+        appIntroClickListeners()
+    }
+
+    override fun onDonePressed(currentFragment: Fragment?) {
+        super.onDonePressed(currentFragment)
+        appIntroClickListeners()
+    }
+
+    private fun appIntroClickListeners() {
```

Comment:
Change function name to navigateToMainFragment()

Reply:
ok

Reply:
@rob729 navigateToMainFragment() or navigateToMainActivity()    ...??

Reply:
Sorry, it should be Activity

Reply:
ok